bcda/client/bluebutton.go

```diff
@@ -142,29 +140,29 @@ func (bbc *BlueButtonClient) getData(path string, params url.Values, jobID strin
 
 	req.URL.RawQuery = params.Encode()
 
-	addRequestHeaders(req, reqID)
-
-	go logRequest(req, jobID)
-	resp, err := bbc.httpClient.Do(req)
-	if resp != nil {
-		logResponse(req, resp, jobID)
-	}
-	if err != nil {
-		return "", err
-	}
+	queryID := uuid.NewRandom()
+	addRequestHeaders(req, queryID)
 
-	defer resp.Body.Close()
+	tryCount := 0
+	maxTries := utils.GetEnvInt("BB_REQUEST_MAX_TRIES", 3)
```

Comment:
BB_REQUEST_MAX_TRIES, BB_REQUEST_RETRY_INTERVAL_MS do these need to be env vars on the ops side?? Also do we want to add this to our docker.compose.yml files or are we going away from that?

Reply:
They do need to be in the ops repo once we decide those values, but it won't break anything to not set them yet because there are defaults. Not really necessary in the Docker configs for the same reason.

Reply:
true the defaults would work, but would also like to get some consistency on when we should create separate env vars and when the defaults are okay. This is a bigger discussion topic outside of this particular ticket, would like to hear @msnook opinion on this.

Reply:
I like sticking with the defaults for now.  We have the flexibility of changing if we need to, without having to redeploy new code.